src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadataV2Feed.cs

```diff
@@ -140,6 +140,8 @@ private static Uri GetUriSafe(string url)
 
         public Task<PackageDeprecationMetadata> GetDeprecationMetadataAsync() => Task.FromResult<PackageDeprecationMetadata>(null);
 
+        public IEnumerable<PackageVulnerabilityMetadata> Vulnerabilities { get; } = null; // Vulnerability metadata is not added to the v2 feed.
```

Comment:
We don't know if other server implementations will be more capable than nuget.org's.

```suggestion
        public IEnumerable<PackageVulnerabilityMetadata> Vulnerabilities { get; } = null; // Vulnerability metadata is not added to nuget.org's v2 feed.
```

Alternatively, use `[JsonIgnore]` to make sure that vulnerabilities does not become part of the V2 protocol.